lib/questions.rb

```diff
@@ -63,33 +75,40 @@ def separate_array_into_even_and_odd_numbers(array)
 # e.g. 'bob'. So in the array ['bob', 'radar', 'eat'], there
 # are 2 palindromes (bob and radar), so the method should return 2
 def number_of_elements_that_are_palindromes(array)
+  array.inject(0){|accum, word| (word == word.reverse) ? accum+1 : accum}
```

Comment:
Surrounding space missing for operator '+'.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.